node_modules/col-core/lib/db.js

```diff
@@ -406,6 +406,8 @@ var setUpModel = function(sequelize) {
    * @property {String}       [pdf_url]               The PDF url of the asset
    * @property {Object}       [preview_metadata]      The preview metadata
    * @property {String}       [url]                   The url of the asset
+   * @property {String}       [aws_s3_object_key]     The Object Key of asset stored in AWS S3
+   * @property {String}       [download_url]          The Canvas url used by users to download asset (file)
```

Comment:
Thanks for noticing that we were missing `download_url`, @johncrossman; of course it doesn't have to be a Canvas URL and points to the ets-collabosphere bucket for newer assets.